src/http.rs

```diff
@@ -13,11 +13,7 @@ pub enum HttpResponseError {
 impl IntoResponse for HttpResponseError {
     fn into_response(self) -> Response {
         match self {
-            HttpResponseError::HTTP(sc) => Response::builder()
-                .status(sc)
-                .body(body::boxed(body::Full::from(sc.to_string())))
-                .unwrap(),
-
+            HttpResponseError::HTTP(sc) => (sc, sc.to_string()).into_response(),
```

Comment:
if you use the body construct and use the `Bytes` body from axum. Via `StatusCode::as_str` you can get a zero copy because all status codes have `&'static str` and `Bytes` is smart via `from_static` to know that it doesn't need to heap allocate a buffer that is then passed via ptr to the writev(2) call. It can just pass the ptr directly to the hyper write call.

Reply:
Thank you but I don't understand what that means in this context and how to apply it to improve the current program?